Library/Koala/Koala.swift

```diff
@@ -491,7 +491,6 @@ public final class Koala {
    */
   public func trackDiscovery(params: DiscoveryParams, page: Int) {
     var props = properties(params: params).withAllValuesFrom(["page": page])
-    props["current_variants"] = AppEnvironment.current.config?.abExperimentsArray
```

Comment:
` current_variants` is already being sent as part of the `defaultProperties`, so this was redundant.